morph/config.py

```diff
@@ -57,7 +57,7 @@
     # whether to modify card Due times based on MorphManIndex. does nothing if relevant notes aren't enabled
     'set due based on mmi': True,
     'ignore maturity': False,        # if True, pretends card maturity is always zero
-
+    'ignore suspended leeches': False, # if True, ignore suspended leeches
```

Comment:
Can we add a setting in jcfg_default() and UI in preferencesDialog.py instead of in this config?  This is fragile, and I'd like to eventually remove config.py.

Reply:
Hey, if you could just make this change then I will merge this.

Thanks!